### DISCUSSION TITLE
justify (grid) content

### DISCUSSION THREAD

**Discussion by trya2l** [typst/typst#4280] [Q&A]
Hi,

I am wondering if there is an in-between `length` and `fraction` in typst.

e.g (working on a 2x2 grid, with a rowspan on the 2nd column):  I would like to align the 2nd column content according to the size of the 1st column. In other words, I would like something like `fr` but in the scope of my grid, not the full page.

**Comment by PgBiel** [comment]
Is this similar to what you're looking for? #4000

**Reply by trya2l** [reply]
Yes, I guess. In fact, I'm looking for something with "flexbox capabilities". I want to be able to justify content (around, between, ... hopefully)
